python/pyproject.toml

```diff
@@ -78,6 +80,12 @@ pyarrow = ["pyarrow"]
 snappy = ["python-snappy"]
 python-snappy = ["zstandard"]
 hive = ["thrift"]
+s3fs = ["s3fs"]
```

Comment:
I think we want to install `fsspec` always, and then we have the option to install different implementations `s3fs`, `gcsfs`, `adlfs` etc

Reply:
Added `fsspec` as a dependency 👍